example_integrations/togetherai/pyproject.toml

```
@@ -0,0 +1,26 @@
+[project]
+name = "together-ai-customer-service"
+version = "0.1.0"
+description = "A customer service agent using Together AI API with Cartesia Line"
+requires-python = ">=3.10"
+dependencies = [
+    "cartesia-line",
+    "openai>=1.0.0",
+    "python-dotenv>=1.0.0",
+    "loguru>=0.7.0",
+    "aiohttp>=3.12.0",
+    "uvicorn==0.35.0",
```

```suggestion
    "uvicorn==0.35.0",
    "websockets>=13.0.0",
```

This should fix the missing websockets dependency when running locally with `uv run`

```
WARNING:  No supported WebSocket library detected. Please use "pip install 'uvicorn[standard]'", or install 'websockets' or 'wsproto' manually.
```